trino/trino_test.go

```diff
@@ -1385,3 +1385,29 @@ func TestSlice3TypeConversion(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkQuery(b *testing.B) {
+	c := &Config{
+		ServerURI:         *integrationServerFlag,
+		SessionProperties: map[string]string{"query_priority": "1"},
```

Comment:
what is it for?

Reply:
I'm not sure, I copied this config from other tests :-)